lesson_06/homework_06.py

```diff
@@ -35,6 +62,18 @@
 Введіть натуральне число: 12345
 Сума цифр числа 12345: 15
 """
+print("Task #6:")
+num = input("Add digit: ")
+sumNum = 0
+if not num.isdigit():
+    print("Enter digit!!!")
+elif int(num) == 0:
+    print("Digit must be above 0!")
+else:
+    for i in num:
+        if int(i) % 2 == 0:
```

Comment:
```suggestion
        if int(i) % 2 == 0:
```
Не зрозумів, навіщо ця перевірка - тоді буде рахувати лише кожне друге число.

Reply:
переплутав поняття натурального числа)
дякую